microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,528 @@
+
+<!-- ------------------------------ -->
+
+#### Completed project
+
+A completed version of this Getting Started project is (⚠️ NOT YET ⚠️) available in the **WebView2Samples** repo. You can use the completed project (from the repo or from following the steps below) as a baseline to add more WebView2 functionality and other features to your HoloLens 2 Unity app.
+
+A completed version of this tutorial project is available in the **WebView2Samples** repo:
+
+* Sample name: **HoloLens2_GettingStarted**
+* Repo directory: [HoloLens2_GettingStarted](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/GettingStartedGuides/HoloLens2_GettingStarted)
+* Unity project folder: **HoloLens2GetStartedApp**
+
+Follow the major Step sections in sequence, below.
+
+<!-- ------------------------------ -->
+
+#### About HoloLens 2, Unity, MRTK, and WebView2
+
+**HoloLens 2** is a groundbreaking, untethered holographic device, offering a hands-free and immersive mixed reality experience. It seamlessly overlays digital information onto the real world, rendering high-definition holograms that stay in place and respond naturally to interaction.
+
+**Unity**, a versatile game engine, is a popular choice for creating 3D applications, including mixed reality experiences for HoloLens 2 devices, due to its extensive features and robust capabilities for AR/VR/MR.
+
+The **Mixed Reality Toolkit (MRTK)** is a Microsoft-driven open source project designed to accelerate the development of mixed reality applications in Unity. MRTK provides a collection of scripts, components, and prefabs specifically designed for building performant and intuitive mixed reality experiences on HoloLens 2 devices.
+
+In Unity, the **Microsoft Mixed Reality WebView** plugin can be installed via the **MRTK Feature Tool**. The WebView plugin for Unity simplifies the integration of WebView2 functionality into your HoloLens 2 app by wrapping the WebView2 control and automatically handling rendering and directing input to the WebView2 control. The plugin also manages interop between Unity and WebView2, enabling communication between JavaScript and Unity via messages and events.
+
+By combining HoloLens 2, Unity, MRTK, and WebView2, you can create immersive mixed reality experiences that seamlessly integrate web content.
+
+<!-- ====================================================================== -->
+## Step 1 - Install Unity and Visual Studio
+
+This tutorial assumes you have Unity 2020.3 LTS or later and Visual Studio 2019 version 16.9 or later installed. Visual Studio 2017 isn't supported.
+
+Follow the [Mixed Reality tools installation guide](../../../windows/mixed-reality/develop/install-the-tools?tabs=unity) to set up your Unity and Visual Studio environment for HoloLens 2 development. These resources will guide you through the process of installing and configuring the necessary tools, including Visual Studio and the Universal Windows Platform development workload.
```

Comment:
```suggestion
Follow the [Mixed Reality tools installation guide](/windows/mixed-reality/develop/install-the-tools?tabs=unity) to set up your Unity and Visual Studio environment for HoloLens 2 development. These resources will guide you through the process of installing and configuring the necessary tools, including Visual Studio and the Universal Windows Platform development workload.
```
scenario: linking to doc that's at LMC but not present repo/TOC.
https://learn.microsoft.com/windows/mixed-reality/develop/install-the-tools?tabs=unity

Reply:
url path now reads:
Follow the [Mixed Reality tools installation guide](/windows/mixed-reality/develop/install-the-tools?tabs=unity) to set up your Unity and Visual Studio environment for HoloLens 2 development. These resources will guide you through the process of installing and configuring the necessary tools, including Visual Studio and the Universal Windows Platform development workload.

you can close this thread